src/network/tarcap/packets_handler/test_state.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "dag/dag_block.hpp"
+#include "transaction_manager/transaction.hpp"
+
+namespace taraxa::network::tarcap {
+
+// THIS NEEDS TO BE REMOVED !!!!!!!
+class TestState {
+ public:
+  TestState() {}
```

Comment:
Cannot be this ctor omitted as it is generated automatically ?

Reply:
good point